cli/src/main.rs

```diff
@@ -373,6 +373,15 @@ fn run() -> Result<(), CliError> {
                                         .about("Unset default value for management type"),
                                 ),
                         )
+                        .subcommand(
+                            SubCommand::with_name("get")
```

Comment:
Any reason we are doing "get" here instead of "show"?

Reply:
It probably should be

Reply:
I changed it to show. Also added a commit to change the `node alias get` to `node alias show`, which was in the PR merged yesterday